schemas/Procedures Schema.json

```diff
@@ -30,6 +30,20 @@
             "items": {"$ref": "#/$defs/prep"},
             "uniqueItems": true
         },
```

Comment:
Some funky spacing here

Reply:
examples need updating - I will work on that soon (might not be until tomorrow)

Reply:
Every editor I open this in shows it with non-funky spacing.... why is it funky here?

Reply:
found it

Reply:
looks like tabs vs spaces